04_MMU/error-codes.yaml

```diff
@@ -326,8 +326,8 @@ Errors:
   approved: true
 
 - code: "04504"
-  title: "FW UPDATE NEEDED"
-  text: "The MMU reports its FW version incompatible with the printer's firmware. Make sure the MMU firmware is up to date."
+  title: "MMU FW UPDATE NEEDED"
+  text: "The MMU firmware version incompatible with the printer's FW. Update to compatible version."
```

Comment:
`The MMU firmware version incompatible with the printer's FW`
missing a verb in this sentence. And there should be those %d.%d.%d placeholders for the real version.

Reply:
Resolved in the latest commit